src/App.jsx

```diff
@@ -1,98 +1,12 @@
-/* eslint-disable max-len */
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import moviesFromServer from './api/movies.json';
+import { MovieList } from './components/MovieList';
```

Comment:
The imported component 'MovieList' does not match the task description 'MoviesList'. Please, correct the name of the imported component from 'MovieList' to 'MoviesList'.